### DISCUSSION TITLE
大家可能不知道的螢幕截圖方式

### DISCUSSION THREAD

**Discussion by pekoranoinu** [1101-GS4719-C/Python-Community-GS4719-C#385] [General]
**按住windows鍵+shift+s**
![未命名](https://user-images.githubusercontent.com/99986220/156324736-f2969c1d-1003-40d9-b82e-419de44f68ef.png)
螢幕會出現**↓**
![11](https://user-images.githubusercontent.com/99986220/156325599-5c4a853e-95d4-4823-b2d2-7264f67064d8.png)
**然後用滑鼠來選取要截圖的畫面就可以搂!**

**Comment by sean1325418** [comment]
原本都是按螢幕最右下角，然後再選螢幕截圖。這方法快多了

**Comment by Ben6uo** [comment]
感謝大哥!我終於不需要用電話拍照再發給別人了!

**Comment by JunTingLin** [comment]
沒錯沒錯💪，這邊整理一下並在補充一個 _截圖方法_
1. 按住windows鍵+shift+s
2. 按ALT + Print Screen (有些不需要按alt)
![image](https://user-images.githubusercontent.com/92431095/156610333-409ecb04-d47d-4562-9162-b69b49a21bf9.png)

第二個的話是直接擷取整個螢幕，到剪貼簿內，所以之後直接按ctrl+v貼上即可

**Reply by 710388** [reply]
這個超級好用

**Comment by ThomasMeng0515** [comment]
原來很多人不知道這個功能，作實驗要擷取螢幕上的圖片或是數據我都是這樣用的!

**Comment by JSHT** [comment]
如果使用 windows 11 的話，可以進入設定 > 協助工具 > 鍵盤 > 使用 PrtScn 按鈕開啟螢幕擷取。就可以透過直接按下 PrtScn 叫出螢幕剪取。
![image](https://user-images.githubusercontent.com/48956859/156709845-23b31643-4f6b-46b4-9ec6-74770a99179e.png)

**Comment by JunTingLin** [comment]
上面講到window內建的螢幕截圖
這邊win也有內建螢幕錄影喔
快速鍵為  win+alt+r
或是按win + G 點選●錄影按鈕
![image](https://user-images.githubusercontent.com/92431095/156896284-92e632b3-f187-44c0-8289-29c7c9bea90f.png)

**Comment by ghost** [comment]
如果要屏幕錄製也有常用且熱門的OBS

**Comment by lx3211** [comment]
MacOS也有快捷鍵喔
shift+command+3截圖
shift+command+4可選擇範圍截圖
shift+command+5可選擇範圍截圖或錄影

**Comment by bloodtrail0511** [comment]
我還真不知道有這個方法，長知識了!一直以來我都是截圖整個螢幕，再去裁切，有夠不方便ㄉ..

**Comment by HuanYuuuuuu** [comment]
這個真是太方便了 現在想要截一小部分就依靠這個了

**Comment by PoooYen** [comment]
printscreen複製貼上也可以喔~~~~

**Comment by SamTU0222** [comment]
這個功能真是太方便了

**Comment by leon10456** [comment]
終於不用再被朋友傳genius的梗圖了~~

**Comment by cordydeng** [comment]
剛好有需要電腦螢幕截圖 , 分享的東西真的太實用了

**Comment by HaoFish123** [comment]
謝謝分享，這方法真好用

**Comment by HuanYuuuuuu** [comment]
這個功能真的超級方便 以前還要特別去截圖.....

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，使用起來很方便。

**Comment by jason79461385** [comment]
從大一用到現在呢！真的是非常方便。

**Comment by alexthebigpp** [comment]
好方便喔，我以前都是從開始裡面找剪取工具，有夠麻煩

**Comment by yenwei12138** [comment]
補充一下，這樣截圖後，畫面是存在剪貼簿裡，如果要把截下來的圖存取下來，可以用內建的小畫家儲存成jpg檔

**Comment by yucheng1028** [comment]
終於找到截圖的快速案法了

**Comment by Lai27330** [comment]
最近線上課程常常需要截各種圖上傳到eeclass，用這些快捷鍵真的方便很多，不必再另外開小畫家或是word檔了。

**Comment by stan890425** [comment]
局部截圖就可以不用事後再修圖了

**Comment by sys511613** [comment]
之前都是把剪取工具定選到工作列，需要的時候就要去點開，但這樣用的可以直接省掉很多步驟，學到了

**Comment by etin2468** [comment]
筆電有功能鍵，好像沒這個困擾...

**Comment by Lynn12378** [comment]
感謝分享，另外有些筆電的f11也可以直接呼叫截圖工具喔

**Comment by 110409538** [comment]
快捷鍵很方便

**Comment by Jimmy911218** [comment]
謝謝分享一開始不知道真的很艱難

**Comment by poiu6fi** [comment]
我之前都用gerena內建的截圖

**Comment by hbh10601** [comment]
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/106755149/a6c1e136-488a-40f0-8119-cfeaaddff1dc)
其實還有這個，剪取工具，也可以直接在上面做編輯、剪裁或存檔等等的動作，我個人都是這個跟Win+Shift+S交互用

**Comment by 710388** [comment]
普普通通沒有到很好用的方法

**Comment by tshsu04224** [comment]
或許大家不知道，我也來分享一個iPhone的截圖功能，很長的頁面其實可以不用分很多張截，可以一次使用長截圖功能就好，按下截圖後，左下角截圖的畫面，按下整頁的選項，就可以把整個頁面一次截下來了!
[image](https://github.com/NCU-GS4719-Python/Python-Community/assets/160737817/f997fb4e-38b9-4439-b83b-f04897e06a7d)

**Comment by cassy0201** [comment]
收到一個方便的功能了！但筆電和桌電好像也有printscreen的按鍵可以直接截圖,也很方便!

**Comment by 155Julia** [comment]
我是mac用戶，這對我完全沒幫助，但我可以分享一下mac是如何截圖甚至螢幕錄影的，cmd+shift+3/4/5分別是截圖整個螢幕、擷取所選畫面、錄製所選或整個螢幕（還能選擇要不要用麥克風），個人覺得超方便，推薦給mac用戶。

**Comment by 155Julia** [comment]
如果有原本windows想換電腦的朋友也超級歡迎換mac，因為它每個功能都可以全部自己重新定義要用什麼快捷鍵去呼叫，超級人性化、超方便，我超喜歡的！

**Comment by stone971275** [comment]
我都是用windows的附屬應用程式中的剪取工具

**Comment by yubo0oo** [comment]
感謝大哥
我活了年終於會螢幕截圖了
我之前一直都用手機拍螢幕
真D感謝你

**Comment by YeeDaniel** [comment]
感謝大大，我學費了
